12-rest/classworks/Alexey_Goremykin/server.js

```diff
@@ -0,0 +1,98 @@
+const Koa = require('koa');
+const Router = require('koa-router');
+const bodyParser = require('koa-bodyparser');
+const jwt = require('jsonwebtoken');
+const koajwt = require('koa-jwt');
+const axios = require('axios');
+const fs = require('fs');
+
+const app = new Koa();
+const publicRouter = new Router();
+const privateRouter = new Router();
+const secret = 'ololo';
+let token = '';
+
+const db = {};
+
+app.use(bodyParser())
+
+publicRouter.post('/signup', (ctx, next) => {
+  let user = ctx.request.body.login;
+  db[user] = ctx.request.body.password;
+  console.log(db);
+})
+
+publicRouter.post('/signin', (ctx, next) => {
+  let login = ctx.request.body.login;
+  for(let key in db){
+    if(key === login){
+      if(db[key] === ctx.request.body.password){
+        token = jwt.sign(ctx.request.body.password, secret); 
```

Comment:
no, no, no, we can't store secure data in JWT token as it's free to be read by anyone who can access it. store only user identificator (login)